guides/common/modules/ref_operating-system-requirements.adoc

```diff
@@ -1,37 +1,47 @@
 :_mod-docs-content-type: REFERENCE
 
-[id="supported-operating-systems_{context}"]
-= Supported operating systems
+[id="operating-system-requirements"]
+= Operating system requirements
 
-ifdef::satellite[]
-You can install the operating system from a disc, local ISO image, Kickstart, or any other method that Red{nbsp}Hat supports.
-Red{nbsp}Hat {ProductName} is supported on the latest version of {RHEL} 9 that is available at the time when {ProductName} is installed.
-Previous versions of {RHEL} including EUS or z-stream are not supported.
-endif::[]
+The following operating systems are supported for deploying {Project}:
 
-The following operating systems are supported by the installer, have packages, and are tested for deploying {Project}:
-
-.Operating systems supported by {foreman-installer}
+.Operating systems supported for deploying {Project}
+[options="header"]
 |====
-| Operating System | Architecture | Notes
+| Operating system | Architecture
 ifdef::foreman-el,katello,orcharhino[]
-| {PlanningDocURL}Enterprise_Linux[{EL}] 9 | x86_64 only | EPEL is not supported.
+| {EL} 9 | x86_64 only
 endif::[]
 ifdef::satellite[]
-| {RHEL} 9 | x86_64 only |
+| {RHEL} 9 | x86_64 only
 endif::[]
```

Comment:
`{EL}` expands to `{RHEL}` on Satellite so this can be
```suggestion
ifdef::foreman-el,katello,orcharhino,satellite[]
| {EL} 9 | x86_64 only
endif::[]
```

Or even:
```suggestion
ifndef::foreman-deb[]
| {EL} 9 | x86_64 only
endif::[]
```

Reply:
Updated here and in the quickstart module.